### DISCUSSION TITLE
Z offset and auto level tools missing when compiling firmware

### DISCUSSION THREAD

**Discussion by Amos343** [Jyers/Marlin#143] [Q&A] [answered]
When compiling firmware from jyers source, the z offset tool is missing from 

Control>Advanced

And the autolevel tool is replaced by "info" on the mainscreen

I'm new to editing firmware and trying to wrap my head around things

Any help would be greatly appreciated

**Comment by RUtarah** [comment]
not missing but you need to uncomment that autolevel section

**Comment by Schnabulation** [comment, marked as answer]
I think you need to activate at least one leveling option in `configuration.h` - be it automatically using BL Touch or manual mesh leveling...
